sunpy/map/sources/stereo.py

```diff
@@ -35,11 +35,9 @@ class EUVIMap(GenericMap):
     def __init__(self, data, header, **kwargs):
 
         GenericMap.__init__(self, data, header, **kwargs)
-
         self._nickname = "{0}-{1}".format(self.detector, self.observatory[-1])
         self.plot_settings['cmap'] = cm.get_cmap('sohoeit{wl:d}'.format(wl=int(self.wavelength.value)))
-        self.plot_settings['norm'] = ImageNormalize(stretch=source_stretch(self.meta, PowerStretch(0.25)))
-        self.meta['waveunit'] = 'Angstrom'
```

Comment:
Same as the above comment about EIT. The `waveunit` keyword should NOT be removed here.